jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamodb/editor/DynamoDbVirtualFile.kt

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.dynamodb.editor
+
+import com.intellij.testFramework.LightVirtualFile
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+
+/**
+ * Light virtual file to represent a dynamo table, used to open the custom editor
+ */
+class DynamoDbVirtualFile(private val tableArn: String, val dynamoDbClient: DynamoDbClient) : LightVirtualFile(tableArn) {
+    init {
+        isWritable = false
```

Comment:
was this supposed to be an override instead?

Reply:
This works, but override is probably safer. Will switch